### DISCUSSION TITLE
Use data buckets in response rules

### DISCUSSION THREAD

**Discussion by Jerantony** [mockoon/mockoon#986] [Ideas] [answered]
It would be great if one could use the data buckets in response rules, so that one can configure for example route parameters which trigger a certain response on the fly. In my case, in a test, I want to receive a 400 response for a route parameter which is randomly generated during the test.

Maybe this could be done either by allowing the data helper in the value of response rules or by including all configured data buckets in comparison operator list of the rule and then using the value as the object path.

**Comment by 255kb** [comment, marked as answer]
I think it would make sense to allow templating helpers in response rules as they are already available in the headers input fields.

**Reply by Jerantony** [reply]
Should I open an issue?

**Reply by 255kb** [reply]
Sorry, I think I forgot to finish my previous message! 
I opened an issue already based on this discussion https://github.com/mockoon/mockoon/issues/988
